Manatee.Trello/Rest/WebApiClient.cs

```diff
@@ -176,11 +177,23 @@ private static HttpContent GetContent(WebApiRestRequest request)
 					formData.Add(content, $"\"{parameter.Key}\"");
 				}
 
-				var byteContent = new ByteArrayContent(request.File);
-				formData.Add(byteContent, "\"file\"", $"\"{request.FileName}\"");
-				TrelloConfiguration.Log.Debug($"\tContent: {formData}");
-
-				return formData;
+                if (File.Exists(request.FilePath))
+                {
+                    var fileStream = File.Open(request.FilePath, FileMode.Open);
+                    var fileInfo = new FileInfo(request.FilePath);
+                    formData.Add(new StreamContent(fileStream), "\"file\"", string.Format("\"{0}\"", "as" + fileInfo.Extension));
+                    formData.Add(new StringContent("mimeType"), "image/png");
```

Comment:
I'd still like to have some basic mime type checking rather than always assuming png.